### PR TITLE
Add io.github.Figma_Linux.figma_linux

### PR DESCRIPTION
**Pull request by ChugunovRoman**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by ChugunovRoman** [comment]
Maybe close [PR#2239](https://github.com/flathub/flathub/pull/2239)

**Comment by ChugunovRoman** [comment]
bot, build io.github.Figma_Linux.figma_linux

File: io.github.Figma_Linux.figma_linux.desktop [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+[Desktop Entry]
+Name=figma-linux
+Comment=Unofficial desktop application for linux
```
Comment (barthalion) [inline io.github.Figma_Linux.figma_linux.desktop:4]:
```suggestion
Comment=Unofficial Figma application for Linux
```

File: io.github.Figma_Linux.figma_linux.desktop [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+Exec="figma-linux --no-sandbox --enable-oop-rasterization --ignore-gpu-blacklist -enable-experimental-canvas-features --enable-accelerated-2d-canvas --force-gpu-rasterization --enable-fast-unload --enable-accelerated-vpx-decode=3 --enable-tcp-fastopen --javascript-harmony --enable-checker-imaging --v8-cache-options=code --v8-cache-strategies-for-cache-storage=aggressive --enable-zero-copy --ui-enable-zero-copy --enable-native-gpu-memory-buffers --enable-webgl-image-chromium --enable-accelerated-video --enable-gpu-rasterization" %U
```
Comment (barthalion) [inline io.github.Figma_Linux.figma_linux.desktop:5]:
```suggestion
Exec=figma-linux --enable-oop-rasterization --ignore-gpu-blacklist -enable-experimental-canvas-features --enable-accelerated-2d-canvas --force-gpu-rasterization --enable-fast-unload --enable-accelerated-vpx-decode=3 --enable-tcp-fastopen --javascript-harmony --enable-checker-imaging --v8-cache-options=code --v8-cache-strategies-for-cache-storage=aggressive --enable-zero-copy --ui-enable-zero-copy --enable-native-gpu-memory-buffers --enable-webgl-image-chromium --enable-accelerated-video --enable-gpu-rasterization %U
```

File: io.github.Figma_Linux.figma_linux.appdata.xml [hunk 3]
```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (barthalion) [inline io.github.Figma_Linux.figma_linux.appdata.xml:1]: Ideally you should maintain it in upstream repo.

File: io.github.Figma_Linux.figma_linux.desktop [hunk 4]
```diff
@@ -0,0 +1,11 @@
+
+[Desktop Entry]
```
Comment (barthalion) [inline io.github.Figma_Linux.figma_linux.desktop:2]: Ideally you should maintain it in upstream repo.

File: io.github.Figma_Linux.figma_linux.desktop [hunk 5]
```diff
@@ -0,0 +1,11 @@
+
```
Comment (barthalion) [inline io.github.Figma_Linux.figma_linux.desktop:1]:
```suggestion
```

File: io.github.Figma_Linux.figma_linux.desktop (hunk 4 above)
Comment (ChugunovRoman) [reply io.github.Figma_Linux.figma_linux.desktop:1]: Yes, sure

File: io.github.Figma_Linux.figma_linux.appdata.xml (hunk 3 above)
Comment (ChugunovRoman) [reply io.github.Figma_Linux.figma_linux.appdata.xml:1]: Yes, sure

File: io.github.Figma_Linux.figma_linux.desktop (hunk 1 above)
Comment (ChugunovRoman) [reply io.github.Figma_Linux.figma_linux.desktop]: Fixed

File: io.github.Figma_Linux.figma_linux.desktop (hunk 2 above)
Comment (ChugunovRoman) [reply io.github.Figma_Linux.figma_linux.desktop]: Fixed

File: io.github.Figma_Linux.figma_linux.desktop (hunk 5 above)
Comment (ChugunovRoman) [reply io.github.Figma_Linux.figma_linux.desktop]: Fixed

**Comment by barthalion** [comment]
bot, build io.github.Figma_Linux.figma_linux

**Pull request closed without merging**